contracts/contracts/lib/utils.tolk

```diff
@@ -217,16 +185,23 @@ fun Bitmap256.has(self, index: int): bool {
     return (self.bits & (1 << index)) != 0;
 }
 
+// For address snake data
+@inline
+fun countAddresses(data: cell): int {
```

Comment:
I'm not sure why, but replacing this with a direct iter().size() call fails the tests for me